packages/p2p/src/stores/my-ads-store.js

```diff
@@ -332,17 +340,28 @@ export default class MyAdsStore extends BaseStore {
     }
 
     @action.bound
-    restrictLength = (e, handleChange) => {
+    restrictLength = (e, handleChange, max_characters = 15) => {
         // typing more than 15 characters will break the layout
         // max doesn't disable typing, so we will use this to restrict length
-        const max_characters = 15;
         if (e.target.value.length > max_characters) {
             e.target.value = e.target.value.slice(0, max_characters);
             return;
         }
         handleChange(e);
     };
 
+    @action.bound
+    restrictDecimalPlace = (e, decimal_place, handleChangeCb) => {
```

Comment:
What is `handleChangeCb`?

Reply:
Can we rename this to `handleChangeCallback` since that's clearer?

Reply:
I have renamed